libraries/AP_Scripting/examples/RCIN_test.lua

```diff
@@ -1,12 +1,32 @@
 -- example of getting RC input
 
+local scripting_rc_1 = rc:find_channel_for_option(86)
```

Comment:
I guess we can't make the RC_Channel enum values visible in Lua?

Reply:
We can such as this
https://github.com/ArduPilot/ardupilot/blob/33ac9efe17e078a1b074fc9b0bf03b38737e3d2e/libraries/AP_Scripting/generator/description/bindings.desc#L73
but its just a duplicate, I didn't fancy doing the whole enum and it would be easy to get out of date, I guess the solution is some sort of automatic translation.